core/src/test/java/tech/tablesaw/io/csv/CsvReaderTest.java

```diff
@@ -173,6 +173,64 @@ public void testDataTypeDetection() throws IOException {
     assertArrayEquals(bus_types, columnTypes);
   }
 
+  @Test
+  public void testNumberDetectionRetainZeroDecimals() throws IOException {
+    Reader reader = new FileReader("../data/immunization.csv");
+    CsvReadOptions options =
+        CsvReadOptions.builder(reader).header(true).sample(false).ignoreZeroDecimal(true).build();
```

Comment:
The auto-formatter renders this on one line, originally one line per operation.

Reply:
I would just leave it like this. The autoformatter allows us to not have to worry about formatting .Even if it's slightly suboptimal in some cases I think it's a win to be able to just ignore formatting

Reply:
Got it @benmccann, no problem, thanks.